src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -1482,6 +1493,33 @@ private async Task<List<TabularData>> GetPeopleInControlDetails(Guid application
             return false;
         }
 
+        private string ExtractOrganisationType(JObject applicationData)
+        {
+            return applicationData.GetValue("OrganisationEducationInstitute")?.Value<string>()
+                   ?? applicationData.GetValue("OrganisationPublicBody")?.Value<string>()
+                   ?? applicationData.GetValue("OrganisationTypeMainSupporting")?.Value<string>()
+                   ?? applicationData.GetValue("OrganisationTypeEmployer")?.Value<string>();
+        }
+
+        private string ExtractAddress(Page page)
+        {
+            if (page == null || page.NotRequired) return string.Empty;
+
+            var result = new StringBuilder();
+
+            var answers = page.PageOfAnswers.First().Answers;
+            var i = 0;
+
+            foreach (var answer in answers)
+            {
+                if (i > 0) result.Append(", ");
+                result.Append(answer.Value);
+                i++;
+            }
```

Comment:
Are we able to use string.Join here instead or will it give incorrect results?

Reply:
Done